libs/perl.json

```diff
@@ -11,18 +11,23 @@
+  {"id":"perl.inline","imports":["Inline"],"name":"Inline","repo":"ingydotnet/inline-pm","tags":["perl-otherlangs"],"tech":["interoperability","https://github.com/ingydotnet/inline-pm"],"status":"awaiting-model"},
   {"id":"perl.Dancer","imports":["Dancer"],"name":"Dancer","repo":"PerlDancer/Dancer","tags":["micro-framework"],"tech":["perl-web"],"status":"awaiting-model"},
   {"id":"perl.Dancer2","imports":["Dancer2"],"name":"Dancer2","repo":"PerlDancer/Dancer2","tags":["framework"],"tech":["perl-web"],"status":"awaiting-model"},
   {"id":"perl.Mojolicious","imports":["Mojolicious"],"name":"Mojolicious","repo":"mojolicious/mojo","tags":["framework"],"tech":["perl-web"],"status":"awaiting-model"},
   {"id":"perl.Plack","imports":["Plack"],"name":"Plack","repo":"plack/plack","tags":["toolkit"],"tech":["perl-web"],"status":"awaiting-model"},
   {"id":"perl.www-mechanize","imports":["WWW:Mechanize"],"name":"WWW-Mechanize","repo":"libwww-perl/WWW-Mechanize","tags":["web-browser"],"tech":["perl-web"],"status":"awaiting-model"},
   {"id":"perl.autodie","imports":["autodie"],"name":"autodie","repo":"pjf/autodie","tags":["throw-on-failure"],"tech":["reliable-code"],"status":"awaiting-model"},
+  {"id":"perl.try-tiny","imports":["Try::Tiny"],"name":"Try-Tiny","repo":"p5sagit/Try-Tiny","tags":["trycatch"],"tech":["reliable-code"],"status":"awaiting-model"},
   {"id":"perl.MongoDB","imports":["Mongos"],"name":"MongoDB","repo":"mongodb/mongo-perl-driver","tags":["mongodb"],"tech":["storage"],"status":"awaiting-model"},
   {"id":"perl.dbd-mysql","imports":["DBI"],"name":"DBD::mysql","repo":"perl5-dbi/DBD-mysql","tags":["mysql"],"tech":["storage"],"status":"awaiting-model"},
   {"id":"perl.Test2","imports":["Test2"],"name":"Test2","repo":"Test-More/test-more","tags":["framework"],"tech":["testing"],"status":"awaiting-model"},
+  {"id":"perl.dist-zilla-plugin-test-reportprereqs","imports":["Dist::Zilla::Plugin::Test::ReportPrereqs"],"name":"Dist-Zilla-Plugin-Test-ReportPrereqs","repo":"dagolden/Dist-Zilla-Plugin-Test-ReportPrereqs","tags":["prerequisites"],"tech":["testing"],"status":"awaiting-model"},
   {"id":"perl.test-deep","imports":["Test::Deep"],"name":"Test-Deep","repo":"rjbs/Test-Deep","tags":["deep-comparison"],"tech":["testing"],"status":"awaiting-model"},
   {"id":"perl.test-most","imports":["Test::Most"],"name":"Test-Most","repo":"Ovid/test--most","tags":["testing"],"tech":["testing"],"status":"awaiting-model"},
   {"id":"perl.test-pod","imports":["Test::Pod"],"name":"Test-Pod","repo":"perl-pod/test-pod","tags":["doc-validation"],"tech":["testing"],"status":"awaiting-model"},
+  {"id":"perl.test-requires","imports":["Test::Requires"],"name":"Test-Requires","repo":"tokuhirom/Test-Requires","tags":["perl-testing"],"tech":["testing"],"status":"awaiting-model"},
+  {"id":"perl.test-synopsis","imports":["Test::Synopsis"],"name":"Test-Synopsis","repo":"miyagawa/test-synopsis","tags":["perl-testing"],"tech":["testing"],"status":"awaiting-model"},
```

Comment:
maybe tags == compile-synopsis? 

from their readme:
"...and then make sure the example snippet code in your SYNOPSIS section passes the perl compile check"